easy_format.ml

```diff
@@ -57,12 +65,14 @@ let list = {
 }
 
 type label_param = {
+  label_break: label_break;
   space_after_label : bool;
   indent_after_label : int;
   label_style : style_name option;
 }
 
 let label = {
+  label_break = `Auto;
```

Comment:
Oh and if we want to inherit in an infinite chain, we need to provide something like this:
```ocaml
val inherit_config : ?a:a -> ?b:b -> (?a:a -> ?b:b -> unit -> config) -> (?a:a -> ?b:b -> unit -> config)
```